wiki/versions/1.19/entity_attributes/src/main/java/org/quiltmc/wiki/entity_attributes/JumpBoostCommand.java

```diff
@@ -0,0 +1,50 @@
+package org.quiltmc.wiki.entity_attributes;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.attribute.EntityAttributeInstance;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.server.command.CommandManager;
+import net.minecraft.server.command.ServerCommandSource;
+
+public class JumpBoostCommand {
+
+	public static int add(CommandContext<ServerCommandSource> context) throws CommandSyntaxException {
+		// @start Apply-Direct
+		EntityAttributeInstance instance = context.getSource().getPlayer().getAttributeInstance(AttributesExample.GENERIC_JUMP_BOOST);
```

Comment:
I'd suggest moving `context.getSource().getPlayer()` to a variable, so it doesn't appear in the wiki and can be copied without trouble